packages/squiggle-lang/src/rescript/TypescriptInterface.res

```diff
@@ -31,6 +31,9 @@ let makeSampleSetDist = SampleSetDist.make
 @genType
 let evaluate = Reducer.evaluate
 
+@genType
+let evaluateUsingExternalBindings = Reducer.evaluateUsingExternalBindings
```

Comment:
Woops! This was unintentional. It's harmless though

Reply:
Will this improve the codebase or worsen it? If it's improve, it's not a big deal, if it's worsen, might as well remove.